tools/ete/ete_tree_generator.xml

```diff
@@ -0,0 +1,81 @@
+<tool id="species_tree_generator" name="species tree generator" version="0.1.1">
+    <description>Generates species tree from list of species using NCBI</description>
+     <requirements>
+        <requirement type="package">ete3</requirement>
+    </requirements>
+    <command>
+    <![CDATA[
+python $__tool_directory__/ete_tree_generator.py 
+-s $speciesFile
+#if $output_format.treebest == 'yes'
+    -f 8
+#else
+    -f ${output_format.format_selector}
+#end if
+-t $output_format.treebest
+> $outputFile
+    ]]>
+    </command>
+    <inputs>
+        <param name="speciesFile" type="data" format="text" label="Species file" help="Species file with list one in each line" />        
+        <conditional name="output_format">
+            <param name="treebest" type="select" label="Use in TreeBest" help="Select yes if specie tree to be used in TreeBest">
+                <option value="yes">Yes</option>
+                <option value="no" selected="true">No</option>
+            </param>
+            <when value="no">
+                <param name="format_selector" type="select" label="Output format">
+                    <option value="0" selected="true">0</option>
+                    <option value="1">1</option>
+                    <option value="2">2</option>
+                    <option value="3">3</option>
+                    <option value="4">4</option>
+                    <option value="5">5</option>
+                    <option value="6">6</option>
+                    <option value="7">7</option>
+                    <option value="8">8</option>
+                    <option value="9">9</option>
+                    <option value="100">100</option>
+                </param>
+            </when>
+        </conditional>
+    </inputs>
+    <outputs>
+        <data type="data" format="nhx" name="outputFile" label="${tool.name} on ${on_string}" />
+    </outputs>
+    <tests>
+        <test>
+            <param name="fastaFile" ftype="text" value="species.txt" />
+            <param name="treebest" type="select" value="yes" />
+            <param name="outputFile" ftype="nhx" value="out.nhx" />
+        </test>
+    </tests>
+    <help>
+    <![CDATA[
+
+Species tree generator from list of species for TreeBest.
```

Comment:
I'd remove `for TreeBest`, this seems a pretty useful tool on its own!
I'd say: "Species tree generator from a list of species using ETE Toolkit"